pages/linux/dunstify.md

```diff
@@ -2,6 +2,7 @@
 
 > A notification tool that is an extension of notify-send, but has more features based around dunst.
 > Works with all options that work for notify-send.
+> More information: <https://dunst-project.org/>.
```

Comment:
```suggestion
> More information: <https://wiki.archlinux.org/title/Dunst#Dunstify>.
```
The official website doesn't even mention `dunstify` as far as I have seen. The only useful documentation was on the Arch wiki.